code/modules/admin/bluespacetech.dm

```diff
@@ -57,10 +59,9 @@
 	bst.update_inv_wear_id()
 	bst.regenerate_icons()
 
-	//BSTs have all languages
-	bst.add_language(LANGUAGE_GALCOM)
-	bst.add_language(LANGUAGE_GUTTER)
-	bst.add_language(LANGUAGE_SIGN)
+	//TODO:
```

Comment:
Commented code

Reply:
a todo isn't code, its a reminder for future features. BSTs are for debugging so this is low priority